packages/gatsby-theme-store/src/components/HomePage/SEO/StructuredData.tsx

```diff
@@ -14,7 +15,10 @@ interface Props extends PageProps<unknown> {
 const removeTrailingSlashes = (x: string) =>
   x[x.length - 1] === '/' ? x.slice(0, x.length - 1) : x
 
-const StructuredData: FC<Props> = ({ siteMetadata: { siteUrl } }) => {
+const StructuredData: FC<Props> = () => {
+  const { host } = useLocation()
+  const siteUrl = `https://${host}`
```

Comment:
In localhost do not have https, this is not a problem?

Reply:
No, this is not a problem, since the canonical should always redirect to the most accurately url, in this case https. I've run lighthouse locally and it was ok :)